packages/vast-xml2js/src/helpers/parseFromString.js

```diff
@@ -0,0 +1,13 @@
+import saneError from 'sane-domparser-error';
+
+const parseFromString = (DOMParser, xmlText) => {
+  const parser = new DOMParser();
+  const doc = parser.parseFromString(xmlText, 'application/xml');
```

Comment:
Why not pass directly parseFromString?

Reply:
I believe you replied in the comment above. so I added my replied above too